Surface_mesh_shortest_path/examples/Surface_mesh_shortest_path/shortest_paths.cpp

```diff
@@ -0,0 +1,61 @@
+#include <cstdlib>
+#include <iostream>
+#include <fstream>
+
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+
+#include <CGAL/Random.h>
+#include <CGAL/Surface_mesh.h>
+
+#include <CGAL/Surface_mesh_shortest_path.h>
+
+#include <boost/lexical_cast.hpp>
+
+typedef CGAL::Exact_predicates_inexact_constructions_kernel Kernel;
+typedef CGAL::Surface_mesh<Kernel::Point_3> Mesh;
+typedef CGAL::Surface_mesh_shortest_path_traits<Kernel, Mesh> Traits;
+typedef CGAL::Surface_mesh_shortest_path<Traits> Surface_mesh_shortest_path;
+typedef boost::graph_traits<Mesh> Graph_traits;
+typedef Graph_traits::vertex_iterator vertex_iterator;
+typedef Graph_traits::face_iterator face_iterator;
+
+int main(int argc, char** argv)
+{
+  // read input polyhedron
+  Mesh polyhedron;
```

Comment:
I just unified it to `typedef { Surface_mesh | Polyhedron | OpenMesh } Triangle_mesh;` and called the variable `tmesh` in all examples.  The EPICK/EPECK discussion was just a misunderstanding.